src/main/jbake/content/docs/user/quick-start.adoc

```diff
@@ -314,3 +319,26 @@ On RHEL, CentOS and Fedora the settings can be made persistent in `/etc/security
 ----
 
 where `[your_username]` needs to be replaced by the by the name of the user who starts Hawkular or Cassandra nodes.
+
+=== Unable to resolve realm public key remotely, status = 404
+
+Exception: `ERROR [io.undertow.request] (default task-2) UT005023: Exception handling request to
+/hawkular/accounts/personas/current: java.lang.RuntimeException: Unable to resolve realm public key remotely, status
+= 404`
+
+This happens when the auth server (Keycloak) cannot be reached. Make sure that the authentication server URL is set
+to the correct location of the Keycloak server. This might be wrong if you are using the `-b` option to bind
+Wildfly to a specific IP or if you have specified a port-offset (`-Djboss.socket.binding.port-offset=123`).
+
+The property to verify is called `auth-server-url` and is located on the following files:
+----
+$HAWKULAR_HOME/standalone/configuration/standalone.xml
+$HAWKULAR_HOME/modules/org/hawkular/nest/main/deployments/hawkular-console.war/keycloak.json
```

Comment:
This could be an option. Right now, the keycloak.json is a static file on the console. We might be able to convert it to a servlet, though.